src/man/sssd-ldap-attributes.5.xml

```diff
@@ -661,7 +661,9 @@
                     <listitem>
                         <para>
                             The LDAP attribute that corresponds to
-                            the group name.
+                            the group name. In an environment with nested
+                            groups, this value must be a LDAP attribute
+                            which has a unique name for every group.
```

Comment:
Hi,

I think it would be worth to mention explicitly that this must be true even for non-POSIX groups in the tree of nested groups.

bye,
Sumit

Reply:
Updated, thanks.